app/Community/Actions/ReplaceBackingGameShortcodesWithGameUrlsAction.php

```diff
@@ -79,7 +79,7 @@ public function execute(string $messageBody): string
                 $parentGameId = $conversionMap[$gameId]['parentGameId'];
                 $setId = $conversionMap[$gameId]['setId'];
 
-                return route('game2.show', ['game' => $parentGameId, 'set' => $setId]);
+                return route('game.show', ['game' => $parentGameId, 'set' => $setId]);
```

Comment:
comment on line 21 still references `game2`

Reply:
Ripped all that stuff out in latest.